examples/power_spectra_example.py

```diff
@@ -0,0 +1,43 @@
+"""
+===============================
+POWER SPECTRUM OF A TIMESERIES
+===============================
+
+An example showing how to estimate power spectrum of a TimeSeries
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import pandas as pd 
+from scipy import signal
+
+import sunpy.timeseries
+from sunpy.data.sample import GOES_XRS_TIMESERIES
+
+###############################################################################
+# Let's first load a TimeSeries from sample data
+
+ts = sunpy.timeseries.TimeSeries(GOES_XRS_TIMESERIES)
+
+###############################################################################
+# We now use scipy's periodogram to estimate the power spectra of the first 
+# column of the Timeseries
+
+cols=ts.columns
```

Comment:
Since probably these examples need to be somehow meaningful I would do the following instead of using a variable called cols:
```python
xrays_short = ts.columns[0]
```
and change `cols[0]` for `xrays_short` everywhere. That makes the code more readable.

Reply:
Done .